Week1/Js-exercises/9-log-the-remainder.js

```diff
@@ -0,0 +1,11 @@
+
+let x = 7;
+x = x % 3 ; // new value of x will be 1 as it's remainder when it's divided by three is 1.
+console.log(x); // we will see it is 1.
+
+let y = 21; // Their value would be the same.Because we didn't assign a new value to them. 
+let z = 13;
+
+console.log(y);  /* We can see they still have their initial values.
+                  Because the remainder includes x but not y and z.*/
+console.log(z);
```

Comment:
😄. There was an error in the homework which David corrected this morning. Would you like to revisit?

Reply:
Updated according to the homework updates . Thanks a lot Ysah for your helpful comments .